src/buf.rs

```diff
@@ -262,7 +262,7 @@ impl<'a> From<&'a GrpcSlice> for GrpcByteBuffer {
     /// Create a buffer from the given single slice.
     ///
     /// A buffer, which length is 1, is allocated for the slice.
-    #[allow(clippy::cast_ref_to_mut)]
+    #[allow(invalid_reference_casting)]
```

Comment:
Does it work for old rustc versions?

Reply:
It gives a warning in 1.71 and earlier, whereas `clippy::cast_ref_to_mut` gives a lint error in 1.72 and newer.